libbeat/docs/http-endpoint.asciidoc

```diff
@@ -0,0 +1,149 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/http-endpoint.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+[[http-endpoint]]
+== HTTP Endpoint
+
+experimental[]
+
+{beatname_uc} can expose internal metrics through a HTTP endpoint. For security
+reasons the endpoint is disabled by default.
```

Comment:
If we're going to say this, I think we should provide more info. Is there something users can do to make sure enabling the endpoint does not cause a security issue?